.github/workflows/test_official.yml

```
@@ -29,9 +29,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -W ignore -m pip install --upgrade pip
-          python -W ignore -m pip install -r requirements.txt
-          python -W ignore -m pip install -r requirements-opts.txt
-          python -W ignore -m pip install -r requirements-dev.txt
+          python -W ignore -m pip install .[dev-all]
```

technically we don't need to install docs, so: 
```suggestion
          python -W ignore -m pip install .[all, dev]
```
?